src/components/Search/SearchAutocompleteInput.tsx

```diff
@@ -226,27 +222,20 @@ function SearchAutocompleteInput(
                         onBlur={() => {
                             autocompleteListRef?.current?.updateExternalTextInputFocus(false);
                             focusedSharedValue.set(false);
+
                             onBlur?.();
                         }}
                         isLoading={isSearchingForReports}
-                        ref={ref}
+                        ref={forwardedRef}
                         type="markdown"
                         multiline={false}
                         parser={parser}
                         selection={selection}
+                        shouldShowClearButton={!!value && !isSearchingForReports}
```

Comment:
```suggestion
                        shouldShowClearButton
                        shouldHideClearButton={false}
```

Reply:
> {((!shouldShowClearButton && inputProps.isLoading !== undefined) || (shouldShowClearButton && !!inputProps.isLoading)) && (

In case you find the inconsistency, then above is also an option to cover all cases